Public/Src/Tools/Execution.Analyzer/Analyzers.Core/XLGPlusPlus/BXLInvocationAnalyzer.cs

```diff
@@ -79,13 +81,22 @@ internal sealed class BXLInvocationAnalyzer : Analyzer
         /// </summary>
         public string OutputFilePath;
 
-        public BXLInvocationAnalyzer(AnalysisInput input) : base(input) { }
+        private Stopwatch m_stopWatch;

```

Comment:
>private Stopwatch m_stopWatch; [](start = 8, length = 30)

this looks like some debug/test code - I don't think it should be a part of the codebase

Reply:
I agree, i'll go ahead and remove it

---
In reply to: [314836322](https://github.com/microsoft/BuildXL/pull/724#discussion_r314836322) [](ancestors = 314836322)